packages/now-build-utils/src/detectors/create-react-app-ejected.ts

```diff
@@ -1,16 +1,20 @@
 import { DetectorParameters, DetectorResult } from '../types';
 
 export default async function detectCreateReactAppEjected({
-  fs: { hasDependency },
+  fs: { hasDependency, getDependencyVersion },
 }: DetectorParameters): Promise<DetectorResult> {
   const hasReactDevUtils = await hasDependency('react-dev-utils');
   if (!hasReactDevUtils) {
     return false;
   }
   return {
     buildCommand: 'node scripts/build.js',
```

Comment:
Did you intentionally leave out `getPackageJsonBuildCommand()` here?

Reply:
@styfle yes, because it calls `node scripts/build.js`

Reply:
@TooTallNate then they change their build command to whatever they need. I'm not sure if the ejected create-react-app has a `build` command, so I just left it here, since it calls node directly, but I guess we can just fall back to that as well